web/src/LogPane.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component, createRef } from 'react';
+import Ansi from "ansi-to-react";
+
+type AnsiLineProps = {
+  line: string
+  key: string
```

Comment:
i'm surprised you need to specify key? all react child components have key, even if it's implicit

Reply:
Yep you're right.